README.md

```diff
@@ -29,9 +29,10 @@ Make sure `typeof process === 'undefined'` returns true when using this library.
 Starting from v2, [monaco-editor-webpack-plugin](https://www.npmjs.com/package/monaco-editor-webpack-plugin) can't be used
 
 Here's the alternative for each options:
+
 - `filename`: it can be configured at the webpack level directly
 - `publicPath`: it can be configured at the webpack level or by hands when redistering the worker in `window.MonacoEnvironment`.
-- `languages`: Monaco languages cannot be used anyway. Import vscode language extensions instead (`@codingame/monaco-vscode-xxx-default-extension`).
+- `languages`: Import vscode language extensions (`@codingame/monaco-vscode-xxx-default-extension`) or (`@codingame/@codingame/monaco-vscode-standalone-*`), but please obey that monaco languages can only be used if `themes` and `textmate` service overrides are not configured.
```

Comment:
And the opposite as well though: language VSCode extensions can't be used without those service overrides

Reply:
Yes, but that is already stated in the corresponding section. Should I refine this here, too?

Reply:
If there is a way to say it in a not confusing way, maybe? some redundancy can't hurt in the doc :)

Reply:
I can do confusing for sure, but I will try to make it not confusing. 🤣

Reply:
Done ⬇️